packages/Babylonian-UI.package/BPRectangleChart.class/class/supportedClass.st

```diff
@@ -0,0 +1,5 @@
+support
+supportedClass
+	
+	
+	^ Point class
```

Comment:
Same here should only be point

Reply:
If changed, I have to update BPVisualization > visualizationClassesForValues: traceValues from
`containedClasses allSatisfy: [:aTracedValueClass | aTracedValueClass isKindOf: aSubclass supportedClass]` to `containedClasses allSatisfy: [:aTracedValueClass | aTracedValueClass isKindOf: aSubclass supportedClass class]` and `aSubclass supportedClass class` sounded weird for me. No strong feelings here though, what do you prefer?